reflex/reflex.py

```diff
@@ -255,6 +269,38 @@ def export(
         )
 
 
+db_cli = typer.Typer()
+
+
+@db_cli.command()
+def migrate():
+    """Create or update database schema based on app models or existing migration scripts."""
+    # explicitly running `reflex db migrate` overrides config file value
+    get_config().db_migrate = True
+    prerequisites.get_app()
+    prerequisites.check_schema_up_to_date()
+
+
+@db_cli.command()
+def makemigrations():
+    """Create autogenerated alembic migration scripts."""
+    if not model.has_alembic:
+        console.print(
+            "[red]Alembic is not installed. Run [bold]pip install alembic[/bold] first."
+        )
+        return
+    if not Path(constants.ALEMBIC_CONFIG).exists():
+        console.print(
+            "[red]Alembic is not initialized. Run [bold]python -m alembic init alembic[/bold] first."
+        )
+        return
+    get_config().db_migrate = False
```

Comment:
Off-topic: it would be nice to have a `set_config` function for cases like this (eg. `set_config("db_migrate", False)`).

Reply:
True, that would be cleaner as we do this in a few places

Reply:
filed https://github.com/reflex-dev/reflex/issues/1373 to capture this as part of a larger refactor!